ledger/participant-integration-api/src/test/resources/keys/community.json

```diff
@@ -0,0 +1,6 @@
+{
+  "algorithm": "HmacSHA256",
+  "encoded": {
+    "bytes": [-120, 67, 83, 21, 126, 32, -7, -16, 47, 56, 20, -41, -100, 104, -56, 85, -9, -84, 55, 57, -106, 20, -86, 104, -1, 126, 115, -91, 73, -41, 18, -56]
```

Comment:
I am wondering why it's not base64 encoded string? it looks very strange like this, and not compact enough :)

Reply:
Changed.